site/docs/spark-writes.md

```diff
@@ -171,6 +172,18 @@ WHERE ts >= '2020-05-01 00:00:00' and ts < '2020-06-01 00:00:00'
 
 If the delte filter matches entire partitions of the table, Iceberg will perform a metadata-only delete. If the filter matches individual rows of a table, then Iceberg will rewrite only the affected data files.
 
+### `UPDATE`
+
+Spark 3 added support for `UPDATE` queries to remove data from tables.
```

Comment:
I think this description isn't quite right. It should be
> Spark 3.1 added support for `UPDATE` queries that update matching rows in tables.

It would also be good to add a pointer to `MERGE INTO` for more complex updates.

Reply:
I've fixed it. I'll think about the complex updates for `MERGE INTO`  and add in another patch.

Reply:
Let's do it here while we're editing the `UPDATE` section.

Reply:
@rdblue I have added some updates for `MERGE INTO`, do I have any other omissions?

Reply:
I think you misunderstood what I was suggesting. Instead of changing `MERGE INTO`, I think that the `UPDATE` docs should point the reader to `MERGE INTO` for more complex cases. Something like this with a link:

> For more complex row-level updates based on incoming data, see the section on `MERGE INTO`.

Reply:
Sorry about that. I have updated.